api/e2e/test/router_test.go

```diff
@@ -31,24 +31,24 @@ var _ = DeployedRouterContext("testdata/create_router_std_ensembler_proprietary_
 				Context("POST /v1/predict", func() {
 
 					When("called with {client: {id: 4}} in the payload", func() {
-						It("responds with data from the `treatment-a` route", func() {
+						It("responds with data from the `control` route", func() {
 							routerE.POST("/v1/predict").
 								WithHeaders(defaultPredictHeaders).
 								WithJSON(json.RawMessage(`{"client": {"id": 4}}`)).
 								Expect().
 								Status(http.StatusOK).
-								JSON().Equal(json.RawMessage(`{"version": "treatment-a"}`))
```

Comment:
this was actually flawed previously. "4" calls treatment "control" which has `route_name=treatment-a`

Since the exp config is patched to pair route and treatment consistently, `id=4` will call control and return `control`